packages/core/content-releases/server/src/utils/index.ts

```diff
@@ -25,14 +25,14 @@ export const getPopulatedEntry = async (
 };
 
 export const getEntryValidStatus = async (
```

Comment:
cc @jhoward1994 just in case, just in case you want to handle publishing validations backend side this might be useful.